python/grapher.go

```diff
@@ -37,14 +37,15 @@ func NewGraphContext(unit *unit.SourceUnit) *GraphContext {
 // been installed (via pip or `python setup.py install`).
 func (c *GraphContext) Graph() (*graph.Output, error) {
 	programMode := os.Getenv("IN_DOCKER_CONTAINER") == ""
-	tc, err := toolchain.Lookup("sourcegraph.com/sourcegraph/srclib-python")
-	if err != nil {
-		return nil, err
-	}
 	pipBin := "pip"
 	pythonBin := "python"
 
 	if programMode {
+		tc, err := toolchain.Lookup("sourcegraph.com/sourcegraph/srclib-python")
```

Comment:
The separation was there before. The idea is that docker is already self contained environment so making virtual env in docker is somewhat pointless.

As for that statement I'm using the `Lookup` to find the binary files which are located in toolchains directory, I think CWD is set to the projects directory so how can I get the path to the binary files? I guess I could use something like [this, but meh](http://godoc.org/github.com/kardianos/osext#ExecutableFolder).

Now that I think of it maybe CWD is toolchains bin dir. Will check.